index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Title here</h1>
+      <p>Some paragraph.</p>
+
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        />
+        <figcaption id="img-caption">Some caption goes here.</figcaption>
+      </div>
+
+      <div id="tribute-info">
```

Comment:
I think you can use the `section` tag instead of `div` to wrap this part of your content. This improves your code to be easy to read and to be organized.

Reply:
Please Change it to sound more like a suggestion rather than an order.